indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -50,6 +43,40 @@
                                    data-title="{% trans %}Delete template{% endtrans %}"
                                    data-update="#email-template-manager">
                                 </a>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="i-box-content">
+                    {% trans %}This email will be sent when the abstract is:{% endtrans %}
```

Comment:
I would wrap it in a `<div>` (as I actually did in https://github.com/indico/indico/pull/2644/commits/3e48d57e8a92e05741cc78b9b27382a15e449f05)